### PR TITLE
upgrade bridgetown gem to 0.21.5, replace node-sass with sass

### PR DESCRIPTION
**Pull request by pragmaticsc**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby 2.7.1
```
Comment (pragmaticsc) [inline .tool-versions:1]: satisfies `asdf` package manager

File: Gemfile [hunk 2]
```diff
@@ -16,4 +16,4 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 #
 # Happy Bridgetowning!
 
-gem "bridgetown", "~> 0.16.0"
+gem "bridgetown", "~> 0.21.5"
```
Comment (pragmaticsc) [inline Gemfile:19]: upgrade to latest version

File: package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "file-loader": "^6.0.0",
     "gh-pages": "^3.1.0",
     "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "^4.13.1",
+    "sass": "^1.32.8",
```
Comment (pragmaticsc) [inline package.json:27]: `node-sass` was deprecated

Comment (pragmaticsc) [reply package.json:27]: had issues installing deprecated version on m1 mac

**Comment by andrewmcodes** [comment]
Hey @vandux 👋 I appreciate the PR! Fortunately, the latest version of Bridgetown negates the need for this repo or action since it now comes bundled with the library.

I'd suggest updating the gem and then running `bridgetown new bridgetown-gh-pages-demo -t erb -c gh-pages` or `bin/bridgetown configure gh-pages` in an existing project. 

This repo has served it's use and I am going to go ahead and archive it. You can see an example of a new site with the included action at [andrewmcodes/bridgetown-gh-pages-demo](https://github.com/andrewmcodes/bridgetown-gh-pages-demo). 

Again, thanks so much for the time you put into this PR! Sorry it took so long to get around to it.

**Pull request closed without merging**